news/templates/news/list.html

```diff
@@ -55,7 +55,7 @@
 </table>
 <br />
 <p>
-	<a class="btn btn-primary btn-xs" href="{{ instance|url:'news-add' }}">
+	<a class="aplus-button--primary aplus-button--sm" href="{{ instance|url:'news-add' }}">
```

Comment:
```suggestion
	<a class="aplus-button--default aplus-button--sm" href="{{ instance|url:'news-add' }}">
```

Reply:
Yikes! The invalid `aplus-button--primary` is back after force-pyshing.